sdk/anomalydetector/ai-anomaly-detector/rollup.base.config.js

```diff
@@ -8,6 +8,7 @@ import { terser } from "rollup-plugin-terser";
 import shim from "rollup-plugin-shim";
 import sourcemaps from "rollup-plugin-sourcemaps";
 import viz from "rollup-plugin-visualizer";
+import { openTelemetryNamedExports } from "../../core/core-tracing/rollup.config.shared";
```

Comment:
this kind of deep import feels a little bad to me. I'd rather have a devDep to an internal package so you can cleanly import it (similar to test-recorder)

Reply:
No objection to that, I can look into it.